server/opendp_apps/dataverses/testing/test_views.py

```diff
@@ -17,11 +17,13 @@ class TestDataverseUserView(BaseEndpointTest):
                 'test_opendp_users_01.json',
                 'test_dataverse_user_01.json']
 
-    def test_create(self):
+    def test_create(self, req_mocker):
+        self.set_mock_requests(req_mocker)
```

Comment:
line 22:
- 2 inputs: `user` and `dv_handoff`
- The inputs should query the models by `object_id`, not `id`